prow/cmd/peribolos/main.go

```diff
@@ -812,6 +843,57 @@ func configureOrg(opt options, client github.Client, orgName string, orgConfig o
 	return nil
 }
 
+type repoClient interface {
+	GetRepos(orgName string, isUser bool) ([]github.Repo, error)
+	CreateRepo(owner string, isUser bool, repo github.RepoCreateRequest) (*github.Repo, error)
+}
+
+func newRepoCreateRequest(name string, definition org.Repo) github.RepoCreateRequest {
+	repoCreate := github.RepoCreateRequest{}
+	repoCreate.Name = &name
```

Comment:
Why are these not part of the `struct` literal?

Reply:
It was previously more complicated and I went line-by-line when changing. I'll fix it.